### ISSUE TITLE
 Can you use NfcAdapter.enableReaderMode() to implement this app?

### ISSUE THREAD

**Issue by zhpncu** [ikarus23/MifareClassicTool#381]
Hi ，

Can you use NfcAdapter.enableReaderMode() to implement this app?

Because this method already exists after android 4.4, this method is better for reading the card

https://developer.android.com/reference/android/nfc/NfcAdapter
public void enableReaderMode (Activity activity, 
                NfcAdapter.ReaderCallback callback, 
                int flags, 
                Bundle extras)

**Comment by ikarus23** [comment]
That is a good idea. I always wanted to use this function, but when I started MCT it was compatible all the way down to Android 2.3 which had no `enableReaderMode()`. Since the current minimum is Android 4.4 there should be no issue in using it.

**Comment by zhpncu** [comment]
Yes, but now there are few phones below Android 4.4, the latest to Android 12, I still recommend you see if you can add this function, if it is compatible with Android 4.4 before, you can also use android sdk level to judge it.

**Comment by Yuki1536** [comment]
Without calling that method the phone may not enter Reader Mode at the time touching another phone with card mode.

The opposite phone enter the Reader Mode and my phone at MCT homepage forced to close and enter the card swiping page,
